server/api/crud/alerts.py

```diff
@@ -184,6 +184,8 @@ def process_event(
                 obj=state_obj,
                 active=active,
             )
+        else:
+            logger.debug("The entity of the alert does not match the one in event")
```

Comment:
Is this expected? in your next PR please add some parameters kwargs to log the entities or some other alert/event information